src/tools/scripts/deploy.sh

```diff
@@ -98,10 +98,31 @@ if [ "$(pgrep -f 'python main.py')" ]; then
   pkill -9 python main.py
 fi
 
-#Remove generated chapters (in case new one from other branch in there)
+#Remove generated chapters and e-books (in case new one from other branch in there)
 # Or with "true" to prevent failure if files don't exist
-echo "Removing Generated Chapters"
+echo "Removing Generated Chapters and e-books"
 rm -f ./templates/*/*/chapters/* || true
+rm -f ./static/pdfs/* || true
+
+# Download latest e-books from Github Action artifacts
+echo "Downloading latest e-books from Github Action artifacts"
+# Set your GitHub repository and workflow name
+REPO_OWNER=HTTPArchive
+REPO_NAME=almanac.httparchive.org
+WORKFLOW_ID=predeploy.yml
+# Set your Github personal access token
+# shellcheck disable=SC1091
+source .env
+#read -sp 'Enter your Github personal access token: ' GITHUB_TOKEN
```

Comment:
Let's just get deployers to set a GITHUB_TOKEN env variable themselves and remove these lines.

Can we add this to here some where? https://github.com/HTTPArchive/almanac.httparchive.org/blob/bed8e5eab20f28633dd2040562379023b7628723/src/README.md?plain=1#L321

Should also explain how to generate an access token and the permission it requires (looks to me `public_repo`).

Also might need to update these steps: https://github.com/HTTPArchive/almanac.httparchive.org/blob/bed8e5eab20f28633dd2040562379023b7628723/src/README.md?plain=1#L359-L372